### ISSUE TITLE
Suggestion for pacing of FAC7

### ISSUE THREAD

**Issue by sofer** [FAC6/FAC7#11]
Given that people were pretty exhausted on FAC6 by the end of week 12 (and before), maybe we should have a one-week break earlier in the course and then space between MVP projects.

How about the following?

Week 1-9: projects
Week 10 : Break
Week 11-12: First round of MVPs
Week 14-15: Second round of MVPs

**Comment by hdrdavies** [comment]
@sofer completely agree! Maybe week 9 is the break?

**Comment by nelsonic** [comment]
@sofer might be worth checking the calendar for when there is a "holiday" during the next cohort and coinciding the break with that? e.g in March 2016 easter Friday and Monday are on 25<sup>th</sup> and 28<sup>th</sup> respectively.  see: http://www.timeanddate.com/calendar/?year=2016&country=9

**Comment by iteles** [comment]
👍🏻👍🏻 To what @nelsonic says. 
Coincidentally, the week of 25th March is week 10 of the course. 👏🏻

**Comment by nelsonic** [comment]
Would a long weekend be enough or do you think people will benefit from a week off? @hdrdavies

**Comment by iteles** [comment]
@nelsonic At the last stop go continue people thought a big reason that people were missing days in week 12 was to take care of 'life admin' before the MVPs started, so we probably need some week days in there as well (banking, any dealings with the council, doctor's appointments, etc).

**Comment by hdrdavies** [comment]
I would say week off to get all those things done Ines is talking about. It seemed from the discussion that people would take a lot of this time to go over things we'd covered in the first 8 weeks anyway so I think a week would help the next cohort a lot with reinforcing material too.

**Comment by sofer** [comment]
@iteles @hdrdavies @nelsonic Given that Easter weekend comes at the end of Week 10, let's do the following:

Weeks 1-9: Jan 18-Mar 18. 9 projects in 9 weeks.
Week 10: Mar 21-24. Short week (MVP prep workshop at the beginning of the week + hackathon?)
Week 11: Mar 29-Apr 1. Short week (MVPs start mid-week)
Weeks 12-13: Apr 4-15. MVP Round 1. Short break before MVP Round 2.
Weeks 14-15: Apr 18-29. MVP Round 2.
Week 16: May 3-6. Short week. Tidy up MVP Round 2. CV building. Moving onto new projects.

**Comment by iteles** [comment]
@sofer I think what @hdrdavies was saying was that some working week days would be appreciated in the break for life admin tasks (boiler men don't come out on the Easter bank holidays and banks are closed)

**Comment by sofer** [comment]
@iteles agreed. e.g. if we were to do a Mon/Tue on week 11 and Wed/Thu/Fri on week 12 that would give people a full week off around Easter.

**Comment by hdrdavies** [comment]
@sofer This seems like a good compromise :+1:

**Comment by iteles** [comment]
Question: Does this render both weeks 'less useful' rather than just one? i.e. If we have a 3 day week and 2 day week, are we better off than having a 4 day week and a week off?

I'm perfectly happy either way, I just don't know the answer to that question!

**Comment by hdrdavies** [comment]
@iteles that's a very good point and I would agree. @sofer what do you think about this? As one of FAC7 is asking me about when their days off will be.

**Comment by sofer** [comment]
@hdrdavies @iteles let's end on March 24th and restart on April 4th.

**Comment by iteles** [comment]
@sofer April 4th is a Saturday :wink: April 6th?

**Comment by hdrdavies** [comment]
@iteles April 4th is a Monday!

**Comment by iteles** [comment]
This would explain why I thought my calendar was going crazy ***_facepalm**_*
<img width="819" alt="screen shot 2016-01-06 at 16 28 26" src="https://cloud.githubusercontent.com/assets/4185328/12147792/9d38f524-b492-11e5-9bec-8da15d6d99f2.png">
